tt1-blocks/functions.php

```diff
@@ -12,14 +12,14 @@
 	 *
 	 * @return void
 	 */
-	function twenty_twenty_one_blocks_setup() {
+	function tt1_blocks_setup() {
 		/*
 		 * Make theme available for translation.
 		 * Translations can be filed in the /languages/ directory.
 		 * If you're building a theme based on Twenty Twenty-One, use a find and replace
```

Comment:
```suggestion
		 * If you're building a theme based on TT1 Blocks, use a find and replace
```

Reply:
If everything else seems fine and since is already approved, @carolinan  do you mind committing your suggestion so as we can merge? GB changes PR is ready as well and will be merged right after this one.

Reply:
Just need to test run it quickly.